### PR TITLE
add SYMLINK_CONDA to planned instructions; test

### PR DESCRIPTION
**Pull request by msarahan**

### CODE REVIEW

**Comment by kalefranz** [comment]
You know I want to get rid of this altogether... https://github.com/conda/conda/issues/2502

**Comment by msarahan** [comment]
Well then, go for it.  I'm only fixing a bug and maintaining things as they
are.  FWIW, the new activate scripts don't scrub the old PATH any more.  I
think you still probably want these symlinks and shortcuts, though, because
without them, if people do not have some part of the root env permanently
on PATH, then the activate scripts need to be available somehow.

File: tests/test_create.py [hunk 1]
```diff
@@ -467,3 +467,17 @@ def test_shortcut_absent_does_not_barf_on_uninstall():
...
         # make sure that cleanup does not barf trying to remove non-existent shortcuts
         check_call(["conda", "remove", '-y', '-p', join(tmp, 'conda'), "console_shortcut"])
+
+
+def test_symlinks_created_with_env():
+    bindir = 'Scripts' if on_win else 'bin'
+
+    with TemporaryDirectory() as tmp:
+        check_call(["conda", "create", '-y', '-p', join(tmp, 'conda'), "python=2.7"])
+        assert isfile(join(tmp, 'conda', bindir, 'activate'))
```
Comment (quasiben) [inline tests/test_create.py]: Suggest `islink` instead of `isfile`

**Comment by kalefranz** [comment]
> if people do not have some part of the root env permanently on PATH, then the activate scripts need to be available somehow.

I think you're basically saying we'd want to symlink always, anyway. And it's a good point.  I agree.

File: tests/test_create.py (hunk 1 above)
Comment (kalefranz) [inline tests/test_create.py]: I'll fix that.

Comment (msarahan) [inline tests/test_create.py]: islink does not work on windows, unfortunately.

**Comment by msarahan** [comment]
Thanks for the merge.

File: tests/test_create.py (hunk 1 above)
Comment (quasiben) [inline tests/test_create.py]: Ah, right.

**Pull request merged**